triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -131,7 +137,7 @@ object TriggerRunnerImpl {
               logger.info(s"Trigger $name is starting")
               running(killSwitch)
             } catch {
-              case cause: Throwable =>
+              case NonFatal(cause) =>
```

Comment:
Boy Scout work: lets not interrupt fatal throwables!

Reply:
:+1: